test/integration/scheduler/queue_test.go

```diff
@@ -190,8 +190,8 @@ func TestCoreResourceEnqueue(t *testing.T) {
 		name string
 		// initialNode is the Node to be created at first.
 		initialNodes []*v1.Node
-		// initialPod is the Pod to be created at first if it's not empty.
-		initialPod *v1.Pod
+		// initialPods is the list of Pods to be created at first if it's not empty.
```

Comment:
nit

```suggestion
		// initialPods is the list of Pods to be created at first if it's not empty.
		// Note that the scheduler won't schedule those Pods,
		// meaning, those Pods should be already scheduled; meaning they should have .spec.nodename.
```

Reply:
Thank you for reviewing.
Updated, could you check this, please?

Reply:
I guess your GitHub shew the old version of my suggestion. Can you update with the latest one?
Sorry, I changed the wording just after posting it so that we can eliminate double `meaning` in the comment.

Reply:
Sorry for bothering you. Updated.